tools/SourceKit/tools/sourcekitd/bin/XPC/Service/XPCService.cpp

```diff
@@ -331,6 +332,11 @@ int main(int argc, const char *argv[]) {
   sourcekitd::initialize();
   sourcekitd::trace::initialize();
 
+  llvm::InitializeAllTargets();
+  llvm::InitializeAllTargetMCs();
+  llvm::InitializeAllAsmPrinters();
+  llvm::InitializeAllAsmParsers();
```

Comment:
This is so tests can work - these normally get called in performFrontend. Clang will barf if we try to emit the PCH without having called these. I assume this is because the object file format is the default?